worlds/alttp/Bosses.py

```diff
@@ -123,7 +123,7 @@ def GanonDefeatRule(state, player: int) -> bool:
     can_hurt = state.has_beam_sword(player)
     common = can_hurt and state.has_fire_source(player)
     # silverless ganon may be needed in anything higher than no glitches
-    if state.world.logic[player] != 'noglitches':
+    if state.world.glitches_required[player] != 0:
```

Comment:
Might want to create `glitchless` and `glitched` properties like the `inverted` property, although this gets used less often so it's not as important for readability.